packages/ui/components/form/date-range-picker/Calendar.tsx

```diff
@@ -11,10 +11,18 @@ import { buttonClasses } from "../../button/Button";
 
 export type CalendarProps = React.ComponentProps<typeof DayPicker>;
 
-function Calendar({ className, classNames, showOutsideDays = true, ...props }: CalendarProps) {
+function Calendar({
+  className,
+  classNames,
+  fromDate,
+  toDate,
```

Comment:
Added fromDate and toDate support to our dateRangePicker which allows us to defined (optionally) which range to enable (disabling dates before `fromDate`, and after `toDate`)